adapters/stroeerCore/stroeercore.go

```diff
@@ -82,6 +87,15 @@ func (a *adapter) MakeBids(bidRequest *openrtb2.BidRequest, requestData *adapter
 			MType: markupType,
 		}
 
+		if bid.DSA != nil {
+			dsaJson, err := json.Marshal(bidExt{bid.DSA})
+			if err != nil {
+				errors = append(errors, err)
```

Comment:
Are you sure if you would like to continue if there is json Marshal error here? If not, then you need to `continue` in your for loop.

Reply:
Yes, I'm sure. The idea is to keep the bid and let upstream decide what to do with it.